syntax = "proto3";

package mrpc;

message RpcHeader
{
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 body_size = 3;
}